src/main.ts

```diff
@@ -14,14 +14,22 @@ async function run(): Promise<void> {
         constants.ACTION_CERTIFICATE_ENCODED,
         constants.ACTION_CERTIFICATE_PASSWORD,
         constants.ACTION_APP_ID,
-        constants.ACTION_TENANT
+        constants.ACTION_TENANT,
+        constants.ACTION_CLI_VERSION
     ]);
 
     try {
         validate(options);
 
-        core.info('ℹ️ Installing CLI for Microsoft 365...');
-        await exec(constants.CLI_NPMINSTALL_COMMAND, [], { silent: true });
+        if (options.CLI_VERSION) {
+            core.info(`☑ Installing CLI for Microsoft 365 (version / tag [${options.CLI_VERSION}])...`);
+            await exec(`${constants.CLI_NPMINSTALL_COMMAND}@${options.CLI_VERSION}`, [], { silent: false });
+        }
+        else {
+            core.info('ℹ️ Installing CLI for Microsoft 365...');
+            await exec(constants.CLI_NPMINSTALL_COMMAND, [], { silent: true });
+        }
```

Comment:
Rather than having this inline, I wonder if it is better to extract this logic into its own function called `getInstallCommand` in the `commands` module, similar to `getLoginCommand`. By moving the logic into its own function, we can add tests to the `commands.spec.ts` to cover the expected output from this command given the options provided.

Reply:
Sounds good to me 👍